front/app/containers/Admin/dashboard/summary/charts/SelectableResourceByTopicChart.tsx

```diff
@@ -65,9 +65,9 @@ class SelectableResourceByTopicChart extends PureComponent<PropsWithHoCs> {
       value: count as number,
       code: topicId as string,
     }));
-    const res = sortBy(mapped, 'name');
 
-    return res.length > 0 ? res : null;
+    const sortedByValue = orderBy(mapped, ['value'], ['desc']);
+    return sortedByValue.length > 0 ? sortedByValue : null;
```

Comment:
Such a pity we have to copy this code :(

Reply:
Indeed, there's probably room for abstraction/standartization with the charts in general. We've taken some first steps and will try to gradually improve as we go :)